_articles/graphics-switch-pop.md

```diff
@@ -42,6 +45,18 @@ Hybrid graphics mode uses both the integrated Intel GPU and the discrete NVIDIA
 Applications will use the integrated GPU unless explicitly requested to use the
 discrete GPU.
 
+Vulkan applications must be launched with the following command to be rendered on the dGPU (NVIDIA):
+
+```
+__NV_PRIME_RENDER_OFFLOAD=1 <application>
+```
+
+GLX applications much be launched with this command before the application to be rendered on the dGPU (NVIDIA):
```

Comment:
Typo, and removing the "before the application."

"GLX applications must be launched with this command to be rendered on the dDPU (NVIDIA):"